### ISSUE TITLE
jsxgettext included in Mailtrain does not support async/await

### ISSUE THREAD

**Issue by bures** [Mailtrain-org/mailtrain#240]
Just stumbled upon that the jsxgettext included in Mailtrain (used to generate pot files) does support async/await. Any ideas?

I believe it might be enough to update it. As @andris9 has patched the version we currently use, thus I would rather leave it up to his expertise.

BTW, I'm trying https://www.i18next.com/ for the client side localization. I've configured it to exploit (using a conversion step) Mailtrain's .mo files.

**Comment by andris9** [comment]
The jsxgettext used by Mailtrain is already a patched version because the original one failed to handle nested handlebars directives (eg `{{#each}}{{#if...`) and threw errors one these. My "fix" was to ignore these errors so it wasn't good enough to push back to upstream but was good enough to handle the translate tags in Mailtrain templates.

One solution would be to keep patching the patched jsxgettext, I can move [the repo](https://github.com/andris9/jsxgettext) under Mailtrain org. Another option would look into alternatives and replace the jsxgettext entirely. I only used it because I had used the same approach in some other projects and it kind of worked, if there's a better approach then it can be used instead.
